### PR TITLE
.github: add feature request template

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR adds a GitHub feature request issue template to our repository.

### Testing procedure

This needs to be merged before one can test it.

### Issues/PRs references

extends #9504

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Feature Request
+about: Ask for missing features or improvements
+
+---
+
+#### Description
+<!-- Here describe what is your need, you use cases and explain why this
```
Comment (miri64) [inline .github/ISSUE_TEMPLATE/feature_request.md]: "Describe here […]", "your use cases"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+feature is important and/or necessary for RIOT. -->
```
Comment (miri64) [inline .github/ISSUE_TEMPLATE/feature_request.md]: "Important and necessary" is somehow redundant IMHO.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (danpetry) [reply .github/ISSUE_TEMPLATE/feature_request.md]: "Please describe here your use case, why you need this feature, and why this feature would be important and/or necessary for RIOT"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+### Useful links
+<!-- Here put links to any documentation that you think is useful -->
+
```
Comment (danpetry) [inline .github/ISSUE_TEMPLATE/feature_request.md]: "Please include links here to any documentation that you think is useful"

**Comment by aabadie** [comment]
Thanks for your quick review @miri64, @danpetry. I pushed a commit addressing your comments.

**Comment by danpetry** [comment]
Me too, please squash

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+about: Ask for missing features or improvements
+
+---
+
+#### Description
+<!-- Please describe your use case, why you need this feature and why this
+feature is important for RIOT. -->
+
+### Useful links
+<!-- Please include links to any documentation that you think is useful -->
```
Comment (danpetry) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Full stop after "useful"

Comment (danpetry) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Seems nit-picky, but these things do have an effect on how professional we seem to contributors...

Comment (danpetry) [reply .github/ISSUE_TEMPLATE/feature_request.md]: After that, ACK from me, please squash and rebase

Comment (aabadie) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Fixed, squashed and rebased.

**Comment by aabadie** [comment]
@danpetry, your last comment is addressed and the branch is squashed. It just needs your ACK and we can go.

**Comment by danpetry** [comment]
ACK.

**Pull request merged**